sycl/include/sycl/platform.hpp

```diff
@@ -194,6 +194,11 @@ class __SYCL_EXPORT platform : public detail::OwnerLessBase<platform> {
   /// \return a vector of all available SYCL platforms.
   static std::vector<platform> get_platforms();
 
+  /// Returns all banned (non-SYCL) platforms in the system.
+  ///
+  /// \return a vector of all banned non-SYCL platforms.
+  static std::vector<platform> get_banned_platforms();
```

Comment:
I would rename "banned" to something neutral. "filtered"? something else?

Reply:
I got inspired by [`IsBannedPlatform`](https://github.com/jchlanda/llvm/blob/jakub/sycl_ls_banned/sycl/source/detail/platform_impl.cpp#L72), but you are right, it does sound quite harsh. Now that I think of it `unsupported` seems right, would that work?